src/main/java/competition/subsystems/drive/commands/DriveToPositionCommand.java

```diff
@@ -3,23 +3,40 @@
 import javax.inject.Inject;
 
 import xbot.common.command.BaseCommand;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
 import competition.subsystems.drive.DriveSubsystem;
 import competition.subsystems.pose.PoseSubsystem;
+import edu.wpi.first.units.measure.Velocity;
 
 public class DriveToPositionCommand extends BaseCommand {
 
     DriveSubsystem drive;
     PoseSubsystem pose;
 
+    double targetPosition;
+    double previousPosition;
+
+    double error;
+    double velocity;
+
+    DoubleProperty proportionalConstant;
+    DoubleProperty derivativeConstant;
+
     @Inject
-    public DriveToPositionCommand(DriveSubsystem driveSubsystem, PoseSubsystem pose) {
+    public DriveToPositionCommand(DriveSubsystem driveSubsystem, PoseSubsystem pose, PropertyFactory propertyFactory) {
         this.drive = driveSubsystem;
         this.pose = pose;
+        propertyFactory.setPrefix("DriveToPositionCommand");
```

Comment:
Command's have a nice way of setting their own name here
```suggestion
        propertyFactory.setPrefix(this);

Reply:
ooh, cool